### PR TITLE
Link data to notations display

### LINKED ISSUE #49: Implement Notation Copy with Data

### PR DESCRIPTION
**Pull request by valarao**
### Overview
This PR resolves #49 and finishes linking the notation to display.

### Other Considerations
- Functionality works, but images need to be fixed; opening a separate issue for that
- Did some minor code clean up (spacing, redundant wrappers)
- Deleted the mock data on the FE, since we no longer need them

### CODE REVIEW

File: client/src/components/dashboard/NotationDisplay.js [hunk 1]
```diff
@@ -2,117 +2,117 @@ import React, { useState } from 'react';
...
-                        <Button className={classes.buttons} variant='outlined' onClick={handleClose}>Cancel</Button>
-                    </Box>    
-                </DialogContent>
-            </Dialog>
+  return (
+    <Dialog className={classes.dialog} onClose={handleClose} open={open}>
+      <DialogTitle className={classes.title}>
+        You Chose To Play
+        <Typography className={classes.name}>
+          {name}
```
Comment (austincho) [inline client/src/components/dashboard/NotationDisplay.js:78]: super small thing but can we bold this? 

image

**Review by austincho — COMMENTED** [review]
Thanks, everything looks good Miguel! I left one small comment.

File: client/src/components/dashboard/NotationDisplay.js (hunk 1 above)
Comment (valarao) [reply client/src/components/dashboard/NotationDisplay.js:78]: Good idea!

**Review by austincho — APPROVED** [review]
Looks great

**Pull request merged**